src/targets/kotlin/api.hbs

```diff
@@ -344,9 +344,14 @@ data class {{{name}}}(
         @SerializedName("{{{key}}}")
     {{/unless}}
     {{/if}}
-        val {{{name}}}: {{#if format}}@Format("{{{format}}}") {{/if}}{{{type}}}{{#unless @last}},{{/unless}}
+    val {{{name}}}: {{#if format}}@Format("{{{format}}}") {{/if}}{{{type}}}{{#unless @last}},{{/unless}}
 {{/each}}
 ) : Serializable {
+{{#each fields}}
+{{#if isNested}}
+    {{> nested }}
```

Comment:
Probably needs the `{{#indent}}` helper here?

Reply:
Rather than supporting arbitrary indentation passed in through a config we should just conform to the code conventions of the target in the actual .hbs file